### PR TITLE
Add Clean Function

### PR DESCRIPTION
**Pull request by t0astt**
This PR adds a `clean(object)` function to the helper file. The `clean` function is designed to iterate over the contents of a JSON object, whether it be a dictionary or list, and recursively clean it.

Why? This will save the developer from having to iterate on their own within their plugin. The developer can simply do something like

```
response = requests.get(...)
some_output = komand.helper.clean(response.json())
```

And be on their merry way. Hopefully this helps to speed up development time.

Tested using mockaroo.com to generate test data, and then serializing/unserializing the data and verifying validity with JSONLint.com

### CODE REVIEW

**Comment by jonschipp** [comment]
@t0astt Excellent. Does this work for Python 2 and 3? master branch is Python 3. Commit it to the 0.2 branch also if it works for Python 2.

File: komand/helper.py [hunk 1]
```diff
@@ -63,6 +63,29 @@ def clean_list(lst):
...
+def clean(object):
+  ''' Returns a new but cleaned JSON object
+  * Recursively iterates through the collection
+  * None type values are removed
+  * Empty string values are removed
+
+  This function is designed so we only return useful data
+  '''
+
+  cleaned = clean_list(object) if isinstance(object, list) else clean_dict(object)
```
Comment (jonschipp) [inline komand/helper.py]: @t0astt Does there need to be a `:` here after the if condition and the else? Doesn't look right to me at first glance

Comment (t0astt) [inline komand/helper.py]: No, there does not. This is the ternary conditional syntax for Python. Looks strange, I know.

Comment (t0astt) [inline komand/helper.py]: It helps to read it as (example): "(Clean the dishes) if [they're dirty], else (leave the dishes alone)."

**Comment by t0astt** [comment]
@jonschipp I believe this was tested in Python 2...? Whatever PyCharm is using as the default. I'll run it against both and make sure they work.

**Comment by jonschipp** [comment]
LGTM Thanks

**Pull request merged**